__tests__/components/editor/property/ResourceList.test.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Stanford University see LICENSE for license
+import React from 'react'
+import { wait } from '@testing-library/react'
+import ResultList from 'components/editor/property/ResourceList'
+import * as sinopiaSearch from 'sinopiaSearch'
+import {
+  renderWithReduxAndRouter, createReduxStore, createBlankState,
+} from 'testUtils'
+
+jest.mock('sinopiaSearch')
+/* eslint import/namespace: 'off' */
```

Comment:
`// eslint-disable-next-line import/namespace`?

Reply:
Changed in latest commit